hydroengine_service/ecopath_functions.py

```diff
@@ -0,0 +1,161 @@
+import ee
+
+import pandas as pd
+
+def submit_ecopath_jobs():
+    # Variable declaration, can be coverted to function args
+    scale = 10000
+    crs = 'EPSG:3035'
+    model = 'HYCOM'
+    str_t_start = '2020-07-01'
+    t_start = ee.Date(str_t_start)
+    t_stop = ee.Date('2021-07-01')
+    bucket = 'slr'
+    # debug = False
+
+
+    extent = [
+        3463000.0001221001148224, # xmin
+        3117163.4356725001707673, # ymin
+        4313000.0001221001148224, # xmax
+        4257163.4356725001707673  # ymax
+    ]
+    geometry =  ee.Geometry.Polygon(
+        coords=[[
+            [extent[0], extent[1]],
+            [extent[0], extent[3]],
+            [extent[2], extent[3]],
+            [extent[2], extent[1]],
+            [extent[0], extent[1]]
+        ]], 
+        proj=crs, 
+        geodesic=False, 
+        maxError=1
+    )
+
+    # Get ImageCollections
+    hycom_currents = ee.ImageCollection("HYCOM/sea_water_velocity")
+    glossis_currents = ee.ImageCollection("projects/dgds-gee/glossis/currents")
+    # hycom_temperature = ee.ImageCollection("HYCOM/sea_temp_salinity")
+
+    # # Settings
+    # palette = ['9EB0FF','93AFFA','87ADF4','79ABED','6CA9E6','60A5DF','54A0D5','489ACA','3E90BC',
```

Comment:
remove this palette, available through https://pypi.org/project/cmcrameri/, but not used